src/Symfony/Component/Translation/Tests/Extractor/PhpExtractorTest.php

```diff
@@ -68,11 +68,15 @@ public function testExtraction($resource)
         $actualCatalogue = $catalogue->all();
 
         $this->assertEquals($expectedCatalogue, $actualCatalogue);
+
+        $filename = implode(\DIRECTORY_SEPARATOR, [__DIR__, '..', 'fixtures', 'extractor', 'translation.html.php']);
+        $this->assertEquals(['sources' => [$filename.':2']], $catalogue->getMetadata('single-quoted key'));
+        $this->assertEquals(['sources' => [$filename.':43']], $catalogue->getMetadata('other-domain-test-no-params-short-array', 'not_messages'));
     }
 
     public function resourcesProvider()
     {
-        $directory = __DIR__.'/../fixtures/extractor/';
+        $directory = implode(\DIRECTORY_SEPARATOR, [__DIR__, '..', 'fixtures', 'extractor']);
```

Comment:
Is this change really needed?

Reply:
You're right. Only the trailing directory separator needs to be removed. Fixed.